marimo/_plugins/ui/_impl/plotly_plot.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2023 Marimo. All rights reserved.
+from __future__ import annotations
+
+import json
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Dict,
+    Final,
+    List,
+    Optional,
+)
+
+from marimo import _loggers
+from marimo._dependencies.dependencies import DependencyManager
+from marimo._output.rich_help import mddoc
+from marimo._plugins.core.web_component import JSONType
+from marimo._plugins.ui._core.ui_element import UIElement
+
+LOGGER = _loggers.marimo_logger()
+
+if TYPE_CHECKING:
+    import plotly.graph_objects as go  # type:ignore
+
+# Selection is a dictionary of the form:
+# {
+#   "points": {
+#     "field1": "value1",
+#     "field2": "value2",
+#   }[],
+#   "range": {
+#     "field1": [min, max],
+#     "field2": [min, max],
+#   },
+# }
+PlotlySelection = Dict[str, JSONType]
+
+
+@mddoc
+class plotly_plot(UIElement[PlotlySelection, List[Dict[str, Any]]]):
+    """Make reactive plots with Plotly Express.
```

Comment:
"Make reactive plots with Plotly", since it works with the `graph_objects` API too